lib/ansible/cli/doc.py

```diff
@@ -369,24 +417,27 @@ def __init__(self, args):
     def tty_ify(cls, text):
 
         # general formatting
-        t = cls._ITALIC.sub(r"`\1'", text)    # I(word) => `word'
-        t = cls._BOLD.sub(r"*\1*", t)         # B(word) => *word*
-        t = cls._MODULE.sub("[" + r"\1" + "]", t)       # M(word) => [word]
-        t = cls._REF.sub(r"\1", t)            # R(word, sphinx-ref) => word
-        t = cls._CONST.sub(r"`\1'", t)        # C(word) => `word'
+        t = cls._ITALIC.sub(_format(r"\1", 'UNDERLINE'), text)  # no ascii code for this
+        t = cls._BOLD.sub(_format(r"\1", 'BOLD'), t)
+
+        # jjj
+        t = cls._CONST.sub(_format(r"`\1'", 'CONSTANT'), t)
+        t = cls._MODULE.sub(_format(r"\1", 'MODULE'), t)    # M(word) => [word]
+        t = cls._REF.sub(_format(r"\1", 'REF'), t)      # R(word, sphinx-ref) => word
+
         t = cls._RULER.sub("\n{0}\n".format("-" * 13), t)   # HORIZONTALLINE => -------
 
         # remove rst
         t = cls._RST_SEEALSO.sub(r"See website for:", t)   # seealso is special and need to break
-        t = cls._RST_NOTE.sub(r"Note:", t)                 # .. note:: to note:
+        t = cls._RST_NOTE.sub(_format(r"Note:", 'bold'), t)  # .. note:: to note:
         t = cls._RST_ROLES.sub(r"website for `", t)        # remove :ref: and other tags
         t = cls._RST_DIRECTIVES.sub(r"", t)                # remove .. stuff:: in general
 
         # handle docsite refs
         # U(word) => word
-        t = re.sub(cls._URL, lambda m: r"%s" % _doclink(m.group(1)), t)
+        t = re.sub(cls._URL, lambda m: _format(r"%s" % _doclink(m.group(1)), 'LINK'), t)
         # L(word, url) => word <url>
-        t = re.sub(cls._LINK, lambda m: r"%s <%s>" % (m.group(1), _doclink(m.group(2))), t)
+        t = re.sub(cls._LINK, lambda m: r"%s <%s>" % (m.group(1), _format(_doclink(m.group(2)), 'LINK')), t)
```

Comment:
Shouldn't the link title be formatted as well, so it's more clear what exactly is the link title?

Reply:
possibly, any recommendations?

Reply:
I'd go for underlining.